tests/unit/specs/pages/ProjectViewEdit.spec.js

```diff
@@ -69,13 +70,12 @@ describe('ProjectViewEdit.vue', () => {
     const propsData = { name: 'local-datashare' }
 
     const wrapper = mount(ProjectViewEdit, { localVue, store, wait, i18n, propsData, config })
-    expect(wrapper.vm.$core.projects).toHaveLength(1)
+    expect(wrapper.vm.$core.projects.length).toBe(1)
```

Comment:
I'm curious, why to you prefer to use `.length`?

Reply:
No idea. I'm changing for `toHaveLength` which is more explicit.